packages/p2p-v2/src/hooks/__tests__/useAdvertiserStats.spec.tsx

```diff
@@ -60,6 +60,8 @@ describe('useAdvertiserStats', () => {
         mockUseSettings.mockReturnValueOnce({
             data: { first_name: 'Jane', last_name: 'Doe' },
         });
+
+        jest.useFakeTimers('modern').setSystemTime(new Date('2024-02-20'));
```

Comment:
Need to set a fixed current time so that testing `daysSinceJoined` does not fail

Reply:
don't we need to restore the timers after?